e2e/session-launcher.test.ts

```diff
@@ -1,23 +1,187 @@
 import {
```

Comment:
I think this filename is not appropriate for these test cases. It should be `session.test.ts' and tests about the session launcher should be included under that file. 

I think we should have separate tests for session creation, and separate tests for the neo session launcher.

e.g.) session.test.ts > create session > user can create... 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/lSyr8xXz1wdXALkJKzVx/719fbf8d-9a87-4eb7-a126-ded5ac7c40c0.png)